Makefile.docker

```diff
@@ -28,7 +28,7 @@ lint:
 	${CDWD} && go fmt ${NO_VENDOR}
 	golint ${SRC}
 
-test:
+test: vendor
```

Comment:
Good point. Maybe I'll just remove this and address it in another PR.  I think you were already doing some makefile changes anyway - so I'll avoid a clash.